perk-store-webapps/src/main/webapp/vue-app/components/perk-store/NoResultFound.vue

```diff
@@ -0,0 +1,82 @@
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <v-flex class="searchNoResults justify-center d-flex border-box-sizing emptyChallenges">
+    <div class="d-flex flex-column">
+      <v-icon
+        size="120"
+        color="primary">
+        {{ icon }}
+      </v-icon>
+      <div class="d-flex flex-column text-center text-sub-title">
+        <span v-if="info" class="text-header-title text-light-color py-5">{{ info }}</span>
+        <span
+          v-if="infoMessage"
+          v-html="emptyPerkstoreSummaryText"
```

Comment:
```suggestion
          v-sanitized-html="emptyPerkstoreSummaryText"
```

Reply:
@boubaker  v-sanitized-html has blocked the propagation of the event, v-html has worked fine !

Reply:
ok, got it, knowing that this is just an I18N and not user provided message, so it's minor. Thanks @ArfaouiMohamedAmine .